SPECS/perl-GSSAPI/perl-GSSAPI.spec

```diff
@@ -8,7 +8,7 @@ Distribution:   Mariner
 
 Name:           perl-GSSAPI
 Version:        0.28
-Release:        31%{?dist}
+Release:        32%{?dist}
```

Comment:
Please replace `%{?_with_testsuite:make test}` in the `%check` section with `make test` and verify the tests succeed. We always have network access during test builds, so Fedora's limitations don't apply.

Reply:
Verified that the tests succeed